src/views/dashboard/data.ts

```diff
@@ -309,7 +309,7 @@ export const CalculationOpts = [
     value: "convertMilliseconds",
   },
   { label: "Seconds to YYYY-MM-DD HH:mm:ss", value: "convertSeconds" },
-  { label: "Precision is 2", value: "precision" },
+  { label: "CPM5D", value: "cpm5d" },
```

Comment:
Hardcoding this label to CPM5D looks unreasonable to me. This calculation type can be selected even if the chart is not `cpm`, for example in the demo video you can select `CPM5D` for the apdex charts. So I'm curious why not use the previous label like `Precision is 5 digits` or so?

Reply:
What do you think of `Decrease 100000 times`? @wu-sheng @kezhenxu94

Reply:
> Hardcoding this label to CPM5D looks unreasonable to me. This calculation type can be selected even if the chart is not cpm, for example in the demo video you can select CPM5D for the apdex charts. So I'm curious why not use the previous label like Precision is 5 digits or so?

This is not a label, this is a formula to format data. Check existing ones, there are things like `apdex`.

Reply:
`Precision is 5 digits` was wrong, I was misguided. `Precision is 5 digits` meant the data is having 5 digits after the dot, but actually OAP doesn't have float, only int/long.

Reply:
We don't have a customizable formula mechanism on UI side, so, from last time, @Fine0830 decided to hard code all possible formulas, then you see what they are today. History story.